pages/pricing.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useState } from 'react'
+import {
+	AiOutlineCheckCircle
+} from 'react-icons/ai'
+import Navbar from '../views/Navbar';
+import Footer from '../components/Footer';
+import Link from 'next/link';
+
+const pricingPlan = [
+	{
+		pricingName: 'Free',
+		duration: '.',
+		pricing: 'for open source projects',
+		features: ['Access of all features', 'Unlimited team size', 'For both Github and Bitbucket'],
+		buttonText: 'Add a Repo',
+		link: 'https://github.com/'
+
+	},
+	{
+		pricingName: 'Standard',
+		duration: 'per user/month',
+		pricing: '$100',
+		features: ['Access of all features', 'personal team support for setting up'],
+		buttonText: 'Start your 30/day trial',
+		link: 'https://chrome.google.com/webstore/detail/vibinex/jafgelpkkkopeaefadkdjcmnicgpcncc' // temp. adding chrome extension link, need to remove it with payment link
+
+	},
+	{
+		pricingName: 'Enterprise',
+		duration: '.',
+		pricing: 'custom pricing',
+		features: ['Good for large teams', 'personal team support for setting up', 'upto 2 custom features'],
+		buttonText: 'Contact Us',
+		link: 'https://api.whatsapp.com/send/?phone=918511557566&text&type=phone_number&app_absent=0'
+
+	},
+
+]
+
+
+const Pricing = () => {
+	const [priceDuration, setPriceDuration] = useState(false); // false for monthly and true for yearly
+	const [pricePlans, setPricePlans] = useState(pricingPlan);
+	const chromeExtensionLink = "https://chrome.google.com/webstore/detail/vibinex/jafgelpkkkopeaefadkdjcmnicgpcncc";
+
+	let heading = [
+		{ name: "Monthly", flag: priceDuration },
+		{ name: "Yearly", flag: !priceDuration },
+	]
+
+	function HandleDuration() {
+		let value = !priceDuration;
+		setPriceDuration(value);
+		if (value) {
+			let features = ['Access of all features', 'personal team support for setting up'];
```

Comment:
?? Why do we need this? Why can't we continue to use the feature list that was defined at the top of this file?

Reply:
it's present in  object so need to make new copy again for the object,